apps/bfd-server/bfd-server-war/src/main/config/server-start.sh

```diff
@@ -53,6 +53,7 @@ visualVm=""
 targetDirectory=
 dbUrl="jdbc:bfd-test:hsqldb:mem"
 v2Enabled="true"
+preadjEnabled="true"
```

Comment:
Will defaulting to true require other changes to prevent enabling it in production?

Reply:
Never mind this isn't used in prod.